docs/drug_metabolite.rst

```diff
@@ -62,42 +66,8 @@ Currently implemented drug metabolite models are:
 | with peripheral compartment(s) | connected compartments.                                  |
 +--------------------------------+----------------------------------------------------------+
 
-~~~~~~~~~~~~~~~~~~~~~
-Structsearch workflow
-~~~~~~~~~~~~~~~~~~~~~
-
-The graph below show how the drug metabolite models are built, with each of the two types 
-of drug metabolite models (basic and pre-systemic) with and without added peripherals.
-For reasons explained above, one of the created candidate models will be chosen as 
-the base model (shown by a square). The base model is chosen as one of the candidate models
-with the fewewst amount of peripheral compartments as possible, with "BASIC" being chosen over
-"PSC". If the inputed model is a drug_metabolite model, this will be used as the base model 
-instead.
-
-.. graphviz::
-
-    digraph BST {
-            node [fontname="Arial"]
-            base [label="Base model"]
-            s1 [label="Base metabolite";shape = rect;]
-            s2 [label="PERIPHERALS(0)"]
-            s3 [label="PERIPHERALS(1)"]
-            s4 [label="Presystemic metabolite"]
-            s5 [label="PERIPHERALS(0)"]
-            s6 [label="PERIPHERALS(1)"]
-
-            base -> s1
-            s1 -> s2
-            s1 -> s3
-            base -> s4
-            s4 -> s5
-            s4 -> s6
-    }
-
 .. note::
-    Peripheral compartments are added using the see :ref:`exhaustive stepwise search algorithm<algorithms_modelsearch>`.
-
-Regarding DVID, DVID=1 is connected to the parent drug while DVID=2 is representing the metabolite.
+    Other types of drug metabolite models are not yet fully supported within the tool.
```

Comment:
What is meant by "not yet fully supported"? We only support models with two DVIDs where DVID=1 is connected to parent drug and DVID=2 is connected to metabolite right? Was thinking this phrasing might be a bit confusing.

Reply:
Yes and the phrasing is a bit confusing. The parsing of drug metabolite models created outside of pharmpy should work, but require the drug metabolite compartment to be named METABOLITE for us to find it currently. And even though we can parse models with, for example, a metabolite conversion <100%, these cannot be created within pharmpy, which is what I meant by "not yet fully supported".

But a note along the lines of

`Drug metabolite models created outside of pharmpy currently require the metabolite compartment to be named 'METABOLITE' in order to function properly`

might fit the description better?

Reply:
Yes, I think that's clearer!